examples/getstarted/src/plugins/mycustomfields/admin/src/components/ColorPicker/ColorPickerIcon/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import Puzzle from '@strapi/icons/Puzzle';
+import icon from './icon.svg';
```

Comment:
Can you use the icons from the DS instead even though it's just an example I think it could be great to ensure best practices regarding icons management.

Reply:
The problem is that only icons in the design system with the right format (the light colored square background) are the default attribute icons, so all custom field developers will have to provide an SVG anyway

In my previous PR I tried to build the background using the Box component, it worked for light mode but was very different from all other icons on dark mode